### PR TITLE
Responsive design for Pocket's Add page

### PR DESCRIPTION
**Pull request by reemhamz**
## Description
The `add` page for pocket wasn't developed to be responsive with it's very (very) rough draft. I'm patching that up now, as well as cleaning up some code and converting the header to use a Split macro.

## Issue / Bugzilla link
https://github.com/mozilla/bedrock/issues/10843
## Testing
http://localhost:8000/en-US/external/pocket/add/

### CODE REVIEW

File: media/css/externalpages/pocket/components/_add.scss [hunk 1]
```diff
@@ -30,49 +41,36 @@ $image-path: '/media/img/externalpages/pocket';
...
-            width: 35%;
+    .mzp-c-button {
+        background-color: $color-button-red;
+        border: 2px solid $color-button-red;
+
+        &:hover,
+        &:focus,
+        &:active {
+            text-decoration: underline!important;
+            background-color: $color-button-red;
```
Comment (reemhamz) [inline media/css/externalpages/pocket/components/_add.scss:52]: Had to copy the styles from the original state to override Protocol `mzp-c-button` styling on the pseudo states.

Comment (nathan-barrett) [reply media/css/externalpages/pocket/components/_add.scss:52]: I think there should be a space between `underline` and `!important`

File: bedrock/externalpages/templates/externalpages/pocket/add.html [hunk 2]
```diff
@@ -18,26 +20,21 @@
...
+          )%}
+          <h2>Save to Pocket from Firefox</h2>
+          <p>Click the Pocket button in your Firefox toolbar to save articles, videos, pages, and more.</p>
+          <a href="https://help.getpocket.com/article/900-saving-to-pocket-and-viewing-your-list-in-firefox" class="mzp-c-button" target="_blank" rel="external noopener">Learn how</a>
+           <p>
+              <a href="https://help.getpocket.com/article/942-where-is-the-pocket-button-in-firefox" class="dark-inline-link rarrow" target="_blank" rel="noopener noreferrer">
+                Don't see the Pocket button?
+              </a>
+          </p>
+          {% endcall %}
```
Comment (nathan-barrett) [inline bedrock/externalpages/templates/externalpages/pocket/add.html:36]: While I love that we're using split here, the layout seems to be a little different from what is on production. I'm going to check in with Dan to make sure he doesn't see any issue with changing it.

**Review by nathan-barrett — CHANGES_REQUESTED** [review]
Looks good overall, but a few changes that are more nitty gritty in nature since I we want to match whats on production pretty closely.

- I think we should update the empty divs with the images that are on production (apple store, google store, etc.). I asked craig if theres an app (like photoshop) that could separate those images for us.
- similarly for the images near the bottom of the page, we should be using the same images from production if we can (the green icon is also partly cut off) since some of them have a border radius
- There are some places where the margins seems a little off compared to production - the biggest one that I saw was the bottom section where the two headers in the columns don't line up.I don't think it needs to be perfect, but if we can do an easy fix than that would be helpful!

File: bedrock/externalpages/templates/externalpages/pocket/add.html (hunk 2 above)
Comment (nathan-barrett) [reply bedrock/externalpages/templates/externalpages/pocket/add.html:36]: Actually just talked with dan and he feels like its fine! So we can keep the split!

**Comment by nathan-barrett** [comment]
Just talked with Craig and he said he would split up the images for us - that way we can use the original images easily. I'll send them to you when I have them!

**Comment by reemhamz** [comment]
@nathan-barrett 
>     * similarly for the images near the bottom of the page, we should be using the same images from production if we can (the green icon is also partly cut off) since some of them have a border radius

I ran this part through Alex last week or so and mentioned to him that some of the apps originally displayed are:
- outdated in images
- the apps aren't used anymore, either with Pocket or just don't exist anymore

So I went with finding the latest icons for some of the apps that make sense to show on the page. The green icon is meant to be cut off, as that is how the company's official icon looks like, and there's not much we can do about that. 
After our conversation we agreed that what we have should suffice for now, especially considering we'll be doing a redesign and updating the apps that Pocket is still compatible with.

File: bedrock/externalpages/templates/externalpages/pocket/add.html (hunk 2 above)
Comment (reemhamz) [reply bedrock/externalpages/templates/externalpages/pocket/add.html:36]: @nathan-barrett glad to hear, the responsiveness of Split was easier to handle for this page than if we didn't use it :)

**Review by nathan-barrett — APPROVED** [review]
🌜 I'll let you merge in case you want you to add anything else.

**Pull request merged**